README.md

```diff
@@ -148,7 +148,17 @@ Video/voice is also super simple! In this example, peer1 sends video to peer2.
 var Peer = require('simple-peer')
 
 // get video/voice stream
-navigator.getUserMedia({ video: true, audio: true }, gotMedia, () => {})
+if (typeof navigator.mediaDevices.getUserMedia === 'undefined') {
```

Comment:
I do not think this check is really needed, we can just switch to new API, old API is too old

Reply:
I know IE 11 is almost dead, but for those few that still need to support it, I would leave it in for another year: https://caniuse.com/#feat=mdn-api_mediadevices_getusermedia

Reply:
Fine. I'll change it. I saw the IE 11 is not supported, but it does not support WebRTC anyway :) https://caniuse.com/#feat=mdn-api_mediadevices_getusermedia